wger/core/views/user.py

```diff
@@ -527,5 +530,6 @@ def get_context_data(self, **kwargs):
                                           _('Name'),
                                           _('Last activity'),
                                           _('Gym')],
-                                 'users': context['object_list']['members']}
+                                 'users': context['object_list']['active_members'],
```

Comment:
@Rhytah I think explicitly using `active_users` instead of just `users` for representing active members would be better.

Reply:
@AGMETEOR I have made the changes